app/src/main/res/layout/fragment_search.xml

```diff
@@ -1,59 +1,74 @@
-<android.support.v4.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    app:layout_behavior="@string/appbar_scrolling_view_behavior"
-    android:orientation="vertical">
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
 
-    <LinearLayout
+    <android.support.v4.widget.NestedScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:orientation="vertical">
+        android:orientation="vertical"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
         <LinearLayout
-            android:id="@+id/search_linear_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:gravity="center"
-            android:layout_margin="@dimen/layout_margin_extra_large"
+            android:layout_height="match_parent"
             android:orientation="vertical">
 
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/go_out"
-                android:gravity="center_horizontal"
-                android:textColor="@color/black"
-                android:layout_marginTop="@dimen/layout_margin_large"
-                android:textSize="@dimen/heading_text_size" />
-
-            <EditText
-                android:id="@+id/location_editText"
+            <LinearLayout
+                android:id="@+id/search_linear_layout"
```

Comment:
Camel case or you can remove it if it is not used anywhere

Reply:
resolved